src/packages/cli/commands/create.js

```diff
@@ -134,9 +135,15 @@ ${green('create')} .gitignore
   });
 
   console.log(`${green('initialize')} git`);
-  console.log('Installing dependencies from npm...');
+  const spinner = new Ora({
+    text: 'Installing dependencies from npm...',
+    spinner: 'dots'
+  });
 
+  spinner.start();
   await exec('npm install', {
     cwd: project
+  }).then(() => {
```

Comment:
Since we use the `await` keyword in this `async function`, calling `.then()` on `exec` is not necessary.

We can rewrite to the following code:

```javascript
spinner.start();

await exec('npm install', {
  cwd: project
});

spinner.stop();
```

--

[Async/Await](https://github.com/tc39/ecmascript-asyncawait)